command-line/lib/Command/Address/Inspect.hs

```diff
@@ -24,6 +28,10 @@ import Cardano.Address.Style.Shelley
     ( inspectShelleyAddress )
 import Control.Applicative
     ( (<|>) )
+import Control.Exception
```

Comment:
Please no implicit imports of modules, this rapidly becomes a mess otherwise :pray:
 
https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#we-use-explicit-imports-by-default-and-favor-qualified-imports-for-ambiguous-functions

Reply:
I wish there was a tool that did that for me.

Reply:
There has been attempts in Haskell. I saw some recent very promising plugins for VSCode using language servers, so hopefully portable to many other editors :crossed_fingers: